### DISCUSSION TITLE
Welcome to laravel-file-viewer Discussions!

### DISCUSSION THREAD

**Discussion by vish4395** [vish4395/laravel-file-viewer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by crazyfella** [comment]
What's the difference? 

$filepath='public/'.$filename;
$file_url=asset('storage/'.$filename);

not working on my end. it's showing 404 error
